rviz_rendering/src/rviz_rendering/render_window.cpp

```diff
@@ -152,18 +154,18 @@ ToString(const EnumType & enumValue)
   return QString("%1::%2").arg(enumName).arg(static_cast<int>(enumValue));
 }
 
+void RenderWindow::resizeEvent(QResizeEvent * resize_event)
```

Comment:
unused parameter

Reply:
so this is a virtual function from the parent QWindow. https://doc.qt.io/qt-5/qwindow.html#resizeEvent

we dont need to use the event param passed in

Reply:
To avoid an unused parameter warning, we can use Q_UNUSED or just write `(void)resize_event`.